uikit/viewmodels/src/main/java/com/sygic/maps/uikit/viewmodels/common/extensions/SdkExtensions.kt

```diff
@@ -173,4 +185,101 @@ fun Application.computePrimaryRoute(routePlan: RoutePlan, routeComputeCallback:
             override fun onPrimaryComputeFinished(router: Router, route: RouteInfo) = routeComputeCallback.invoke(route)
         })
     }
+}
+
+fun List<NaviSignInfo.SignElement>.concat(): String = StringBuilder().apply {
```

Comment:
toto je nejaka zvlastna funkcia... concat hovori, ze nieco spaja, ale toto neberie druhy parameter. Ak to ma robit na itemoch toho listu, tak by to chcelo iny nazov. Napr `concatItems`.

Reply:
A navyse to vzdy zacne ciarkou, to je chcene?

Reply:
alebo to isNotEmpty je metoda buildra? 🔎 Sa to tam pletie.

Reply:
isNotEmpty je metoda builderu. přejmenuji tu metodu na `concatItems`.